src/utility/Lang.ts

```diff
@@ -1837,7 +1837,18 @@ namespace TinyWars.Utility.Lang {
                 `默认为当前可用选项中最靠前的一项。`,
             ].join("\n"),
 
-            `Untranslated...`,
+             [
+                `This option determines your turn order. `,
+                ``,
+                `The turn order this game is listed as below. `,
```

Comment:
```suggestion
                `The turn order of the country in this game is locked, which listed as below. `,
```

Reply:
Imo it is unnecessary, but if you want such a format, it can be something like this: 
The turn order (of the country (unnecessary imo)) in this game is certain, which is listed below.